website/docs/r/monitor_alert_prometheus_rule_group.html.markdown

```diff
@@ -0,0 +1,189 @@
+    for        = "PT5M"
+    severity   = 2
+
+    action {
+      action_group_id = azurerm_monitor_action_group.example.id
+      action_properties = {
+        actionName = "actionValue"
+      }
+    }
+
+    resolve_configuration {
+      auto_resolved   = true
+      time_to_resolve = "PT10M"
+    }
+
+    annotations = {
+      annotationName = "annotationValue"
+    }
+
+    labels = {
+      team = "prod"
+    }
+  }
+  tags = {
+    key = "value"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Alert Management Prometheus Rule Group. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the Azure Region where the Alert Management Prometheus Rule Group should exist. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Alert Management Prometheus Rule Group should exist. Changing this forces a new resource to be created.
+
+* `rule` - (Required) A `rule` block as defined below.
+
+* `scopes` - (Required) Specifies the resource ID of the Azure Monitor Workspace.
+
+* `cluster_name` - (Optional) Specifies the name of the Managed Kubernetes Cluster.
+
+* `description` - (Optional) The description of the Alert Management Prometheus Rule Group.
+
+* `rule_group_enabled` - (Optional) Is this Alert Management Prometheus Rule Group enabled? Possible values are `true` and `false`.
+
+* `interval` - (Optional) Specifies the interval in which to run the Alert Management Prometheus Rule Group represented in ISO 8601 duration format. Possible values are between `PT1M` and `PT15M`.
+
+* `tags` - (Optional) A mapping of tags to assign to the Alert Management Prometheus Rule Group.
+
+---
+
+A `rule` block supports the following:
+
+* `action` - (Optional) An `action` block as defined below.
+
+* `alert` - (Optional) Specifies the Alert rule name.
+
+* `annotations` - (Optional) Specifies a set of informational labels that can be used to store longer additional information such as alert descriptions or runbook links.
+
+* `enabled` - (Optional) Is this rule enabled? Possible values are `true` and `false`.
+
+* `expression` - (Required) Specifies the Prometheus Query Language expression to evaluate. For more details see [this doc](https://prometheus.io/docs/prometheus/latest/querying/basics). Evaluate at the period given by `interval` and record the result as a new set of time series with the metric name given by `record`. 
+
+* `for` - (Optional) Specifies the amount of time alert must be active before firing, represented in ISO 8601 duration format.
+
+* `labels` - (Optional) Specifies the labels to add or overwrite before storing the result.
+
+* `record` - (Optional) Specifies the recorded metrics name.
+
+* `resolve_configuration` - (Optional) A `resolve_configuration` block as defined below.
```

Comment:
this might make more sense as\ ?
```suggestion
* `alert_resolution` - (Optional) A `alert_resolution` block as defined below.
```

Reply:
Thanks for your feedback. I have updated the code. Could you please take another look?